silk/tests/openthread/ot_test_form_network.py

```diff
@@ -94,6 +94,13 @@ def test01_Pairing(self):
         for end_node in self.joiner_list:
             end_node.whitelist_node(self.router)
             self.router.whitelist_node(end_node)
+        
+        # whitelisting nodes 1 - 6 for full mesh
+        mesh_nodes = self.device_list[:min(6, len(self.device_list) - 1)]
```

Comment:
I wonder why `-1` is necessary. If you need to get the first 6 nodes of `device_list`, you can just use `device_list[:6]`. It works for `device_list` with less than 6 elements.  
```suggestion
        mesh_nodes = self.device_list[:6]
```